### PR TITLE
Disable selection for highlight meta

### LINKED ISSUE #448: How to disable selection for Dollar signs in Markdown code blocks
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [x] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Actual behavior 

In common, if you `select all` in the code block, the `$` will be included.

screen shot 2018-10-21 at 1 25 09 pm

### Expected behavior 

screen shot 2018-10-21 at 1 24 22 pm

demo link: https://almostover.ru/2016-10/centos-ftp-server-install/

How can I get the style like @ivan-nginx?

### PR DESCRIPTION
**Pull request by stevenjoezhang**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [x] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [x] Tests for the changes was maked (for bug fixes / features).
   - [x] Muse | Mist have been tested.
   - [x] Pisces | Gemini have been tested.
- [ ] [Docs](https://github.com/theme-next/theme-next.org/tree/source/source/docs) in [NexT website](https://theme-next.org/docs/) have been added / updated (for features).

## PR Type
**What kind of change does this PR introduce?**

- [ ] Bugfix.
- [x] Feature.
- [ ] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [ ] Other... Please describe:

## What is the current behavior?

Issue resolved: #448

## What is the new behavior?

- Screenshots with this changes: N/A
- Link to demo site with this changes: N/A

### How to use?
In NexT `_config.yml`:
```yml
...
```

## Does this PR introduce a breaking change?
- [ ] Yes.
- [x] No.

### CODE REVIEW

File: source/css/_common/components/highlight/highlight.styl [hunk 1]
```diff
@@ -110,14 +107,24 @@ pre {
...
+    background: $highlight-deletion;
+  }
+
+  .addition {
+    background: $highlight-addition;
+  }
 
-  .comment { color: $highlight-comment; }
+  .meta {
+    color: $highlight-yellow;
```
Comment (1v9) [inline source/css/_common/components/highlight/highlight.styl]: Why change color?

File: source/css/_common/components/highlight/highlight.styl [hunk 2]
```diff
@@ -110,14 +107,24 @@ pre {
...
+    disable-user-select();
```
Comment (1v9) [inline source/css/_common/components/highlight/highlight.styl]: Only for shell lang, bash doesn't have `meta` class

Comment (1v9) [reply source/css/_common/components/highlight/highlight.styl]: Otherwise we need to remind users to use shell for $

**Comment by 1v9** [comment]
And `codeblock.copy_button` still copy meta, maybe add a option or do something like refactoring `dimmer` by you before --- don't give choice to users 😉

File: source/css/_common/components/highlight/highlight.styl (hunk 2 above)
Comment (stevenjoezhang) [reply source/css/_common/components/highlight/highlight.styl]: https://github.com/theme-next/hexo-theme-next/issues/448

File: source/css/_common/components/highlight/highlight.styl (hunk 1 above)
Comment (stevenjoezhang) [reply source/css/_common/components/highlight/highlight.styl]: Duplicate of function color

**Review by 1v9 — APPROVED** [review]

**Comment by stevenjoezhang** [comment]
Relevant issue https://github.com/theme-next/hexo-theme-next/issues/1110

**Comment by stevenjoezhang** [comment]
Reverted in https://github.com/next-theme/hexo-theme-next/commit/9fdaba295a2c6c707a7d96d331762ab571b89c1a

markdownlint recommends that don't use dollar signs before commands without showing output.

https://github.com/DavidAnson/markdownlint/blob/main/doc/md014.md

**Pull request merged**